src/main/java/net/greatstart/services/UserService.java

```diff
@@ -18,11 +19,13 @@
 
     private UserDao userDao;
     private RoleService roleService;
+    private PasswordTokenDao passwordTokenDao;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "passwordTokenDao" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)